### PR TITLE
[#9382] Migrate student feedback session results page to Angular

### PR DESCRIPTION
**Pull request by wkurniawan07**
Part of #9382 

- Only text and contribution question types are migrated so far
- Feedback response comments are not supported yet
- Question statistics are not supported yet, so contribution question result will still display nothing for students since they cannot see individual result for that question type

### CODE REVIEW

File: src/web/app/pages-session/session-result-page/session-result-page.component.ts [hunk 1]
```diff
@@ -10,9 +22,40 @@ import { Component, OnInit } from '@angular/core';
...
+      this.httpRequestService.get('/session', paramMap).subscribe((resp: FeedbackSession) => {
+        const TIME_FORMAT: string = 'ddd, DD MMM, YYYY, hh:mm A zz';
+        this.session = resp;
+        this.formattedSessionOpeningTime =
+            moment(this.session.submissionStartTimestamp).tz(this.session.timeZone).format(TIME_FORMAT);
+        this.formattedSessionClosingTime =
+              moment(this.session.submissionEndTimestamp).tz(this.session.timeZone).format(TIME_FORMAT);
+        this.httpRequestService.get('/result', paramMap).subscribe((resp2: SessionResults) => {
+          this.questions = resp2.questions.sort(
+              (a: QuestionOutput, b: QuestionOutput) => a.questionNumber - b.questionNumber);
```
[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/web/app/pages-session/session-result-page/session-result-page.component.ts]: Are the question numbers normalized in the session submission page?

[the remark this answers is not in the document]
Comment (xpdavid) [reply src/web/app/pages-session/session-result-page/session-result-page.component.ts]: Yup. https://github.com/TEAMMATES/teammates/blob/master/src/main/java/teammates/ui/webapi/output/FeedbackQuestionsData.java#L25

[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/web/app/pages-session/session-result-page/session-result-page.component.ts]: @xpdavid Is this case possible:
- Questions 1, 2, 4 are visible and answerable by students -> they are normalized to 1-3
- Question 3 (actual, not normalized) is for instructor, but the response can be viewed by students
- When viewing responses, the students can see all questions 1-4, but this does not tally with the normalized 1-3 in the session submission page

[the remark this answers is not in the document]
Comment (xpdavid) [reply src/web/app/pages-session/session-result-page/session-result-page.component.ts]: I see. So we should remove the normalisation for submission page?

**Pull request merged**